lib/repository.php

```diff
@@ -55,7 +55,12 @@ public static function add($entity, $save = true) {
 
     public static function edit($id, $entity) {
         if (array_key_exists($id, self::$entities)) {
-            self::$entities[$id] = array_merge(self::$entities[$id], $entity);
+            if (is_array(self::$entities[$id])) {
+                $data =  array_merge(self::$entities[$id], $entity);;
```

Comment:
Please fix the double semicolon at the end.

Reply:
fixed